doc/whats_new.rst

```diff
@@ -46,7 +46,7 @@ Requirements
 Bug fixes
 ^^^^^^^^^
 
-- xxx
+- Anonymization of EDF files in :func:`write_raw_bids` will now convert recording date to ``01-01-1985 00:00:00``, while setting the recording date in the ``scans.tsv`` file to an anonymized date < 1925, thus making the file EDF/EDFBrowser compliant, by `Adam Li`_ (:gh:`669`)
```

Comment:
In fact the user has to specify the time shift in days to move the date into the correct, BIDS-compatible region (1900–1925 IIRC). So this is **not** done automatically.